CMake/sitkLanguageOptions.cmake

```diff
@@ -42,13 +58,12 @@ endmacro()
 #-----------------------------------------------------------
 # Lua
 
-set_QUIET( WRAP_LUA )
-find_package ( Lua ${_QUIET} )
+set_DEFAULT_QUIET( WRAP_LUA )
 
-if ( LUA_FOUND )
-  set( WRAP_LUA_DEFAULT ${WRAP_DEFAULT} )
-else()
-  set( WRAP_LUA_DEFAULT OFF )
+if( NOT DEFINED WRAP_LUA_DEFAULT OR WRAP_LUA )
+  find_package ( Lua ${_QUIET} )
+
+  set( WRAP_LUA_DEFAULT ${LUA_FOUND} )
 endif()
 
 set( LUA_ADDITIONAL_LIBRARIES "" CACHE STRING "Additional libraries which may be needed for lua such as readline.")
```

Comment:
Look like this call can be moved within the `if(WRAP_LUA)`, I don't think defining the variable `LUA_ADDITIONAL_LIBRARIES` makes sense otherwise